contracts/cw1-subkeys/src/multitest/tests.rs

```diff
@@ -31,18 +30,20 @@ macro_rules! assert_sorted_eq {
 
 #[test]
 fn get_contract_version_works() {
-    let mut app = App::default();
+    let app = App::default();
 
-    let owner = Addr::unchecked("owner");
+    let owner = "owner";
 
-    let code_id = Cw1SubkeysCodeId::store_code(&mut app);
+    let code_id = Cw1SubkeysContractCodeId::store_code(&app);
 
     let contract = code_id
-        .instantiate(&mut app, &owner, &[], true, "Sublist contract")
+        .instantiate(vec![owner.to_owned()], true)
+        .with_label("Sublist contract")
+        .call(owner)
         .unwrap();
 
     let version: ContractVersion =
-        query_contract_info(&app.wrap(), contract.addr().clone()).unwrap();
+        query_contract_info(&app.app.borrow().wrap(), contract.contract_addr.to_string()).unwrap();
```

Comment:
The `sylvia::App` should have some of those functions exposed - in particular `wrap()` so it is callable directly.

Reply:
unfortunately we cannot expose wrap through sylvia::App. borrow().wrap() will try to return reference to local value. We can however hide the borrow behind the app() and app_mut() or something like that